packages/react-wildcat-prefetch/README.md

```diff
@@ -0,0 +1,69 @@
+<img src="http://static.nfl.com/static/content/public/static/img/logos/nfl-engineering-light.svg" width="300" />
+
+# react-wildcat-prefetch
+
+[![npm package](https://img.shields.io/npm/v/react-wildcat-prefetch.svg?style=flat-square)](https://www.npmjs.org/package/react-wildcat-prefetch)
+
+A React decorator to prefetch component data.
+
+## Installation
+
+jspm:
+
+```bash
+jspm install react-wildcat-prefetch
+```
+
+npm:
+
+```bash
+npm install react-wildcat-prefetch
+```
+
+## Usage
+
+## `@prefetch({string}|{function} action, [{Object} options])`
+
+```js
+// Prefetch data from a URL
+@prefetch("https://example.com/data.json")
+
+// Or pass in a function
+function getData() { /** **/ }
+@prefetch(getData)
+
+// By default the key name is `asyncData`, but you
+// can set your own key name in the options object
+@prefetch("/superbowls.json", {key: "newKeyName"})
```

Comment:
Prefetch requires a fully qualified URL for server-side use. Can you update this example?